app/controllers/finders_controller.rb

```diff
@@ -12,6 +12,8 @@ class FindersController < ApplicationController
   def show
     respond_to do |format|
       format.html do
+        raise UnsupportedContentItem unless content_item.is_search? || content_item.is_finder?
```

Comment:
I think the only page on GOV.UK with a `search` content item is /search, which doesn't hit this controller.  So this condition can just be `unless content_item.is_finder?`

Reply:
@barrucadu thanks, I've changed that.